idd/Energy+.idd.in

```diff
@@ -312,24 +313,30 @@
 !
 !      kPa                    =>   inHg                0.29523
 !      m                      =>   in                  39.3700787401575
+!      m3/hr                  =>   gal/hr              264.172037284185
+!      m3/hr-m2               =>   gal/hr-ft2          24.5423853466941
+!      m3/hr-person           =>   gal/hr-person       264.172037284185
+!      m3/m2                  =>   gal/ft2             24.5423853466941
+!      m3/person              =>   gal/person          264.172037284185
 !      m3/s                   =>   gal/min             15850.3222370511
+!      m3/s-m                 =>   gal/min-ft          4831.17821785317
+!      m3/s-W                 =>   gal-hr/min-Btu      4645.27137336702
```

Comment:
Ok, I will also change:
  m2-K/W                 =>   ft2-F-hr/Btu
to 
  m2-K/W                 =>   (ft2-F)/(Btu/hr)